cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -173,12 +174,14 @@ Feature: Users can authneticate with OIDC authenticator
     Then it is bad gateway
 
   Scenario: Performance test
+    Given I skip
```

Comment:
We've changed this test to not include the "max" part. I suggest to not skip it so we can see if the issue is resolved